### PR TITLE
chore: Bump version, update example project dependency.

### PR DESCRIPTION
**Pull request by mjhenkes**
- Closes 

### User facing changelog

N/A

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/example/package.json [hunk 1]
```diff
@@ -17,18 +16,14 @@
...
-  ],
   "dependencies": {
     "bluebird": "3.5.3",
     "glob": "7.1.3"
   },
   "devDependencies": {
     "chai": "3.5.0",
     "cross-env": "6.0.3",
-    "cypress-example-kitchensink": "https://github.com/cypress-io/cypress-example-kitchensink.git#994a943b76f1186c10ce083f48266cee2f76aeb4",
+    "cypress-example-kitchensink": "https://github.com/cypress-io/cypress-example-kitchensink.git#bd93f68568bc0a412b2c4dd2ddbc3b7cb89e5799",
```
Comment (mjhenkes) [inline packages/example/package.json]: crumbs, do i need to release a 12.0.1 for this?

Comment (emilyrohrbough) [reply packages/example/package.json]: I don't think so. its a dev dependencies & isn't shipped with the examples.

File: packages/example/package.json [hunk 2]
```diff
@@ -37,5 +32,10 @@
     "resolve-pkg": "2.0.0",
     "rimraf": "3.0.2",
     "shelljs": "0.8.5"
-  }
+  },
+  "files": [
+    "cypress",
+    "lib"
+  ],
+  "types": "index.d.ts"
```
Comment (emilyrohrbough) [inline packages/example/package.json:40]: why did we need this change?

**Review by emilyrohrbough — APPROVED** [review]

File: packages/example/package.json (hunk 1 above)
Comment (jennifer-shehane) [reply packages/example/package.json]: It is what scaffolds the kitchensink tests when e2e tests are scaffolded, so it is shipped.

Comment (mjhenkes) [reply packages/example/package.json]: Why is it a dev dependency?

Comment (jennifer-shehane) [reply packages/example/package.json]: Why is this in devDeps though. 🤔

**Review by AtofStryker — APPROVED** [review]

**Pull request merged**